pkg/gadgets/audit-seccomp/tracer/tracer.go

```diff
@@ -28,8 +28,8 @@ import (
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target amd64 -cc clang auditseccomp ./bpf/audit-seccomp.c -- -I./bpf/ -I../../.. -target bpf -D__KERNEL__
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target amd64 -cc clang auditseccompwithfilters ./bpf/audit-seccomp.c -- -DWITH_FILTER=1 -I./bpf/ -I../../.. -target bpf -D__KERNEL__
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target $TARGET -cc clang auditseccomp ./bpf/audit-seccomp.c -- -I./bpf/ -I../../../ -I../../../${TARGET} -target bpf -D__KERNEL__
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target $TARGET -cc clang auditseccompwithfilters ./bpf/audit-seccomp.c -- -DWITH_FILTER=1 -I./bpf/ -I../../../ -I../../../${TARGET} -target bpf -D__KERNEL__
```

Comment:
Is it necessary to add `-I../../../${TARGET}` given that you added `#ifdef __TARGET_ARCH_arm64` in `audit-seccomp.h`.

Same question for other files.

Also, should it be `pkg/vmlinux/arm64` instead of `pkg/arm64/vmlinux`? It seems that only vmlinux has this separation by arch.